minecraft/franchise/multiplayer.go

```diff
@@ -0,0 +1,67 @@
+package franchise
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/sandertv/gophertunnel/minecraft/franchise/internal"
+)
+
+type MultiplayerToken struct {
```

Comment:
Can you document what a multiplayer token is, and how the user should use/create it (i.e. by  calling RequestMultiplayerToken)